paddle/fluid/distributed/fleet_executor/fleet_executor_desc.proto

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto2";
+package paddle.distributed;
+
+message FleetExecutorDesc {
+	required string grain = 1 [default = "coarse"];
+	optional string machine = 2;
```

Comment:
What does the keyword `machine` be used for? The ip of current machine? If yes, can this keyword be combined with the keyword `port` below to get another string kind keyword `addr` indicating 'ip:port', which will be used for brpc?